### ISSUE TITLE
Tagpicker component does not take aria properties.

### ISSUE THREAD

**Issue by nipunja** [OfficeDev/office-ui-fabric-react#2457]
Make tagpicker component accessbile to assisstive technologies.

**Comment by kristoferbrown** [comment]
TagPicker and other pickers can now have aria properties passed in via the newly added IInputProps interface. Currently it only allows aria-label to be passed in, but this can be easily expanded in the future if appropriate.

**Comment by mdmdakbar** [comment]
Hi, what we want is to get aria-label suppose 'Tags' when tags is empty but when we have tags inside that suppose we have 3 tags namely 'red','rose','blue' then it should announce the value of tags also like "Tags red rose blue". How can we achieve that?
